roles/openshift_master_facts/lookup_plugins/openshift_master_facts_default_predicates.py

```diff
@@ -53,7 +53,7 @@ def run(self, terms, variables=None, regions_enabled=True, short_version=None,
             short_version = re.sub('^1.', '3.', short_version)
 
         if short_version == 'latest':
-            short_version = '3.6'
+            short_version = '3.7'
```

Comment:
Ok, if we are flipping openshift-ansible to install 3.7 by default then yes, this should be 3.7.